### PR TITLE
Disable click propagation of search element

### PR DESCRIPTION
**Pull request by maettyhawk**
#130

### CODE REVIEW

File: src/leafletControl.js [hunk 1]
```diff
@@ -100,7 +100,8 @@ const Control = {
       const { form } = this.searchElement.elements;
       const root = map.getContainer().querySelector('.leaflet-control-container');
 
-      const container = createElement('div', 'leaflet-control-geosearch bar');
+      const container = L.DomUtil.create('div', 'leaflet-control-geosearch bar');
```
Comment (smeijer) [inline src/leafletControl.js]: I don't like the dependency on `L.DomUtil` here, we should use the `createElement` method as used on other places in the code.

File: src/searchElement.js [hunk 2]
```diff
@@ -3,7 +3,8 @@ import { ESCAPE_KEY, ENTER_KEY } from './constants';
 
 export default class SearchElement {
   constructor({ handleSubmit = () => {}, searchLabel = 'search', classNames = {} } = {}) {
-    const container = createElement('div', ['geosearch', classNames.container].join(' '));
+    const container = L.DomUtil.create('div', ['geosearch', classNames.container].join(' '));
```
Comment (smeijer) [inline src/searchElement.js]: Sorry, `leaflet` dependency in this file is a no-go.

**Comment by smeijer** [comment]
@maettyhawk I've implemented it differently. Can you take a look if this works for you?

Don't get me wrong; I do appreciate your contributions 👍

**Comment by smeijer** [comment]
I decided to publish it with the current release; `v2.5.0`. Please let me know if it works for you, and what you think of the solution.

**Comment by maettyhawk** [comment]
No worries, I'm fine as long as it helps to improve the package, eventhough it was just food for thought in that case. ;-)

**Pull request merged**